src/gamma_inc.jl

```diff
@@ -0,0 +1,599 @@
+using Base.Math: @horner
+#useful constants
+const acc0 = [5.0e-15 , 5.0e-7 , 5.0e-4] #accuracy options
+const big1 = [25.0 , 14.0, 10.0]
+const e0 = [0.25e-3 , 0.25e-1 , 0.14]
+const x0=[31.0 , 17.0 , 9.7]
+const alog10 = log(10)
+const rt2pin = 1.0/sqrt(2*pi)
+const rtpi = sqrt(pi) 
+const exparg = -745.1
+
+wk = zeros(30)
+#---------COEFFICIENTS FOR MINIMAX APPROX.-------------------
+
+a0=[-.231272501940775E-02 , -.335378520024220E-01 , -.159840143443990E+00 , -.333333333333333E+00]
+b0=[.633763414209504E-06 , -.939001940478355E-05 , .239521354917408E-02 , .376245718289389E-01 , .238549219145773E+00 , .729520430331981E+00]
+a1=[-.398783924370770E-05 , -.587926036018402E-03 , -.491687131726920E-02 , -.185185185184291E-02 ]
+b1=[.386325038602125E-02 , .506042559238939E-01 , .283344278023803E+00 , .780110511677243E+00]
+a2=[.669564126155663E-03 , .413359788442192E-02]
+b2=[-.421924263980656E-03 , .650837693041777E-02 , .682034997401259E-01 , .339173452092224E+00 , .810647620703045E+00]
+a3=[.810586158563431E-03 , .649434157619770E-03]
+b3=[-.632276587352120E-03 , .905375887385478E-02 , .906610359762969E-01 , .406288930253881E+00 , .894800593794972E+00]
+a4=[-.105014537920131E-03 , -.861888301199388E-03]
+b4=[.322609381345173E-01 , .178295773562970E+00 , .591353097931237E+00 , .103151890792185E+01]
+a5=[-.435211415445014E-03 , -.336806989710598E-03]
+b5=[.178716720452422E+00 , .600380376956324E+00 , .108515217314415E+01]
+a6=[-.182503596367782E-03 , .531279816209452E-03]
+b6=[.345608222411837E+00 , .770341682526774E+00]
+a7=[.443219646726422E-03 , .344430064306926E-03]
+b7=[.821824741357866E+00 , .115029088777769E+01]
+a8=[.878371203603888E-03 , -.686013280418038E-03]
+
+#----------------COEFFICIENTS FOR TEMME EXPANSION------------------
+
+d00 = -.333333333333333E+00
+d0=[.833333333333333E-01 , -.148148148148148E-01 , .115740740740741E-02 , .352733686067019E-03 , -.178755144032922E-03 , .391926317852244E-04]
+d10 = -.185185185185185E-02
+d1=[-.347222222222222E-02 , .264550264550265E-02 , -.990226337448560E-03 , .205761316872428E-03]
+d20 = .413359788359788E-02
+d2=[-.268132716049383E-02 , .771604938271605E-03]
+d30 = .649434156378601E-03
+d3=[.229472093621399E-03 , -.469189494395256E-03]
+d40 = -.861888290916712E-03
+d4=[.784039221720067E-03]
+d50 = -.336798553366358E-03
+d5=[-.697281375836586E-04]
+d60 = .531307936463992E-03
+d6=[-.592166437353694E-03]
+d70 = .344367606892378E-03
+d80 = -.652623918595309E-03
+
+# The kernel of log1pmx
```

Comment:
maybe move these to another file, and note the source (i.e. copied from StatsFuns.jl)